waiter/src/waiter/process_request.clj

```diff
@@ -611,8 +646,9 @@
                                time (assoc :suspended-at (utils/date-to-str time))
                                (not (str/blank? last-updated-by)) (assoc :last-updated-by last-updated-by))]
       (log/info "Service has been suspended" response-map)
-      (track-response-status-metrics service-id service-description 503)
+      (track-response-status-metrics request 503)
       (meters/mark! (metrics/service-meter service-id "response-rate" "error" "suspended"))
+      (rlog/log-request request :status 503 :termination-state :service-suspended)
```

Comment:
Should we assume that this is a terminal operation for processing the response and trigger `log-request`? 
`process` is in control of what response get written and is better positioned to make the `log-request` call.